cmd/go-swagger/restapi/configure_swagger_bench.go

```diff
@@ -18,12 +18,12 @@ import (
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/sql_injection"
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/ssrf"
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/swagger_server"
-	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/unvalidated_redirect"
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/xss"
 	"github.com/Contrast-Security-OSS/go-test-bench/internal/common"
 )
 
 //go:generate go run ../regen/regen.go
+//go:generate rm -rf ./operations/
```

Comment:
These are in the wrong order. The delete line needs to come first.

Still working on the intermittent missing file failure.

Reply:
I think the intermittent problem I was seeing was caused by a race condition, where `go generate ./...` was looking for additional files to scan for `go:generate` statements, and sometimes tried to access files under `operations/` as they were being deleted. The solution here is to change the scope of the command we're using - to specify the package with generate statements, rather than telling it to look at all packages.

i.e.
```diff
-go generate ./...
+go generate ./cmd/go-swagger/restapi
```